bench/Runtime.hs

```diff
@@ -102,8 +102,8 @@ main = do
 
     --
       vMA = MA.fromList MA.Seq vList :: MA.Array MA.P MA.Ix1 Double
-      aMA = MA.resize' (MA.Ix2 n n) vMA :: MA.Array MA.P MA.Ix2 Double
-      bMA = MA.resize' (MA.Ix2 n n) $ MA.fromList MA.Seq uList :: MA.Array MA.P MA.Ix2 Double
+    aMA' <- MA.resizeM (MA.Sz2 n n) vMA :: IO (MA.Array MA.P MA.Ix2 Double)
```

Comment:
Is there a specific reason for using the IO monad here instead of the `let` binding?

Reply:
I just updated the PR switching back to `resize'`, but that is only to make benchmarks compatible with older version of `massiv-0.2.*`.

It is always better to use `resizeM` vs `resize'` (and all other functions that follow that patterns) cause then you fail earlier, if there is indeed an error. Consider this example:

```haskell
λ> a = (0 :. 0) ... (1 :. 5)
λ> a' = resize' (Sz1 6) a
λ> print (computeAs U a')
Array U *** Exception: SizeElementsMismatchException: (Sz (2 :. 6)) vs (Sz1 6)
λ> a'' <- resizeM (Sz1 6) a
*** Exception: SizeElementsMismatchException: (Sz (2 :. 6)) vs (Sz1 6)
```